students/barecki_andrzej/lesson_02_flow_control/L3_Bishop_moves.py

```diff
@@ -0,0 +1,20 @@
+# In chess, the bishop moves diagonally, any number of squares. Given two different squares of the chessboard,
+# determine whether a bishop can go from the first to the second in one move.
+
+# The program receives as input four numbers from 1 to 8,
+# specifying the column and row numbers of the starting square and the column and row numbers of the ending square.
+# The program should output YES if a Bishop can go from the first square to the second in one move, or NO otherwise.
```

Comment:
możesz używać multiline comments, mniej klikania: https://stackoverflow.com/questions/7696924/way-to-create-multiline-comments-in-python